### ISSUE TITLE
Always export 9:16 media

### ISSUE THREAD

**Issue by mzorz** [Automattic/stories-android#451]
From @megsfulton 👍 

To reduce the amount of variability with hiding parts of the media, one thing we might want to consider is having the mobile editors always output a media size that is 9:16, regardless of the device screen ratio.

Here’s how that might look. I don’t believe it would require any UI changes if we crop from the bottom “safe area” where the slide thumbnails are.

![always-9_16-output-android-editing](https://user-images.githubusercontent.com/6597771/88050746-d07c5f00-cb2d-11ea-94b3-c92cf2bd960f.png)

#### Follow up notes / questions on this from  me:

 [can you confirm] (the bottom part of the screen is what gets cropped, variably depending on the device screen size and adjustments to get a 9:16 out of it).

> MF: That’s correct. To make it extra obvious to the user, we could make the opacity on the bottom bar 100% black on devices taller that 9:16

<img width="445" alt="screen-shot-2020-07-14-at-9 35 35-am" src="https://user-images.githubusercontent.com/6597771/88050755-d3774f80-cb2d-11ea-98ed-d89d422c8ee0.png">

#### Further information gathered

Most of the formats we can hope to be able to produce are 9:16 but there will still be cases where we can only produce 240×320 (4:3) or 480×720 (3:2).
In more detail: the new produced video will be of a different size than the original and this time it’ll be portrait oriented, to include the added panda emoji that sits on the now otherwise blank area in the example. The output video format (including the video frame size) will depend on what the device can do for us (also may or may not be the quality of the original video, for example), and is going to be normalized to these sizes:

```
// MUST
// - 320 x 240 px
// - 720 x 480 px
// SHOULD (when hardware available)
// - 1280 x 720 px
// - 1920 x 1080 px
```

In addition to these 4 up there, we also have these other 2 formats for bigger screens as we saw these are also widely supported nowadays on more recent Android versions:

- 1440p = 2560×1440
- 2160p = 3840×2160

While most of them are 9:16, the only 2 that are marked MUST are not which are 480p (3:2) and 240×320 (4:3)

**Comment by mzorz** [comment]
**Images**
#479 should handle things for images

**Video**
We’re already normalizing video sizes to fit 9:16 as well since https://github.com/Automattic/stories-android/pull/329, using letterboxing (https://github.com/Automattic/stories-android/pull/113) for example as can be seen on [this video](https://mzorz.files.wordpress.com/2020/08/wp_story1596725155032_1.mp4). This was recorded in landscape, fed into the Stories library, added text and then re-recorded in portrait at a resolution of 1080×1920, fitting 9:16. 🎉

**Comment by mzorz** [comment]
Closed in #479 and #481
